src/PDFPasswordForm.tsx

```diff
@@ -0,0 +1,164 @@
+import React, {useState, useRef, useMemo, type ChangeEvent, type FormEventHandler} from 'react';
+import PropTypes from 'prop-types';
+
+import {pdfPasswordFormStyles as styles} from './styles';
+
+type Props = {
+    isPasswordInvalid: boolean;
+    isFocused: boolean;
+    onSubmit: (password: string) => void;
+    onPasswordUpdated: (password: string) => void;
+    onPasswordFieldFocused: (isFocused: boolean) => void;
```

Comment:
```suggestion
    onSubmit?: (password: string) => void;
    onPasswordUpdated?: (password: string) => void;
    onPasswordFieldFocused?: (isFocused: boolean) => void;
```

According to your propTypes.

Reply:
Done - https://github.com/Expensify/react-fast-pdf/commit/9ad9d67cbb120fb481b8ec846b743d0f667a9523.